fvm/src/machine/engine.rs

```diff
@@ -242,8 +263,8 @@ impl Engine {
         }
     }
 
-    /// Load some wasm code into the engine.
-    pub fn load_bytecode(&self, k: &Cid, wasm: &[u8]) -> anyhow::Result<Module> {
+    /// Loads some Wasm code into the engine and prepares it for execution.
+    pub fn prepare_wasm_bytecode(&self, k: &Cid, wasm: &[u8]) -> anyhow::Result<Module> {
```

Comment:
I found `load_bytecode` ambiguous; it sounds like we're loading code from the engine, not _into_ the engine. So I proposed a more unambiguous name here.